steam_app/views/GamesTableViewCell.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  GamesTableViewCell.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 25/01/23.
+//
+
+import UIKit
+
+class GamesTableViewCell: UITableViewCell {
+    static let nibName: String = "GamesTableViewCell"
+    
+    let starFav: UIButton = {
```

Comment:
it's better to make your UI properties private as they should not be accessible outside of the class